lib/improver/cli/apply_emos_coefficients.py

```diff
@@ -329,6 +218,9 @@ def process(current_forecast, coeffs, landsea_mask, distribution,
     calibrated_predictor, calibrated_variance = ac.process(
         current_forecast, coeffs, landsea_mask=landsea_mask)
 
+    if shape_parameters:
+        shape_parameters = [np.float32(x) for x in shape_parameters]
```

Comment:
Is this actually needed?  Presumably the computation is handled in the plugin and it doesn't affect the CLI test outputs?

Reply:
Agreed that it should be handled by the plugin. I couldn't see (at a glance) whether it was or not, so felt safer casting anyway (because the old argparser code would have done that for us). To be honest, there's a huge amount of complexity that could be dropped from this CLI, so I think we'd be better off doing it all in one go.